frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -74,6 +75,8 @@ export const useListPageFilter: UseListPageFilter = require('@console/internal/c
   .useListPageFilter;
 export const ResourceLink: React.FC<ResourceLinkProps> = require('@console/internal/components/utils/resource-link')
   .ResourceLink;
+export const ResourceStatus: React.FC<ResourceStatusProps> = require('@console/dynamic-plugin-sdk/src/app/components/utils/resource-status')
+  .default;
```

Comment:
Just directly export the component... no need to require internally to get the component.